### DISCUSSION TITLE
k8s 部署streampark 无法修改默认h2数据库为mysql或者pg的问题

### DISCUSSION THREAD

**Discussion by yjz-enter** [apache/streampark#3193] [Q&A]
streampark 版本：v2.1.1
flink 版本：1.17.1
部署步骤
git clone https://github.com/apache/incubator-streampark.git
cd incubator-streampark/deploy/helm
helm template streampark/ -n streampark -f streampark/values.yaml --output-dir ./result
kubectl create serviceaccount streampark 
kubectl create clusterrolebinding flink-role-binding-flink --clusterrole=edit --serviceaccount=streampark:streampark
kubectl apply -f result/streampark/templates
下面是我的具体配置文件

application-mysql.yml configmap.yaml
![0a921cf88e012e0b30250f9a224d0ce](https://github.com/apache/incubator-streampark/assets/51939654/361f8a6e-a7a8-40ba-a737-553f2446f830)
![6923546c5c3ac7e9517b72509d890cd](https://github.com/apache/incubator-streampark/assets/51939654/4d58a31b-43e1-4c30-b2ff-31439c251d6a)
![967d0d35425c7e620cc5fa140ec9e2c](https://github.com/apache/incubator-streampark/assets/51939654/efc3802e-e91f-4ad2-8370-edbba5e62dcd)
![img_v2_a0eb1c16-ddbc-4b34-8366-c3708ad6590g](https://github.com/apache/incubator-streampark/assets/51939654/fb274583-f1b7-4842-be9a-b7f135a1a4c5)
这是一张streampark 容器内部/streampark/conf 路径下application.yml 文件 ，可以看到还是h2
![image](https://github.com/apache/incubator-streampark/assets/51939654/2771de32-3ee5-4c96-8285-440a9f2c5229)
这是 容器内部/streampark/conf 路径下application-mysql.yml 文件可以看到并没有修改成功
![image](https://github.com/apache/incubator-streampark/assets/51939654/50fb72b7-bf3f-4e4f-a77d-3658c22e575a)

**Comment by NullHunt** [comment]
这不是streampark的问题, 重启docker容器看看
